sdorfehs.1

```diff
@@ -343,6 +343,18 @@ Spawn a shell executing
 .Ar shell\-command ,
 showing _NET_WM_PID supporting programs in the given frame instead of
 the frame selected when this program is run.
+.It Ic execw Ar shell\-command
+Spawn a shell executing
+.Ar shell\-command ,
+but wait for it to finish.  This should behave the same as
+.Ar exec
+but will wait for completion.  Use timeout(1) if it might hang, or
+.Nm sdorfehs
```

Comment:
I see, I noticed the first occurrence in the file had the name, so I had just copied that.   Must be that it remembers the first one, rather than the most recent one, since `Nm .` does not reprogram the default with `.` but just appends the period.